proposals/lip-0048.md

```diff
@@ -52,6 +52,7 @@ We define the following constants:
 | `LENGTH_TOKEN_ID`                    | uint32 | 8                                      | The length of a token ID in bytes.                                |
 | **Configurable Constants**           |        | **Mainchain Value**                    |                                                                   |
 | `MIN_FEE_PER_BYTE`                   | uint64 | 1000                                   | Minimum amount of fee per byte required for transaction validity. |
+| `ZERO_FEE_PER_BYTE_MAX_BLOCK_HEIGHT`                   | uint32 | 0                                   | Block height ending the `MIN_FEE_PER_BYTE = 0` initial period. |
```

Comment:
Yes, I suppose it would be better to reorder the wording of that constant, to make it more readable. Also, I didn't really have a nice name for it. If you have any further suggestions on how to name it, I woul appreciate it.
Regarding the fact that the mainchain value is set to `0`: on the mainchain, the fee-free period we're introducing does not exist. So, `MIN_FEE_PER_BYTE` should be set to mainchain default value (`= 1000 Beddows`) from the start, i.e., from the genesis block. So, I was thinking that setting `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE = 0` as a mainchain value would do the trick. Am I right?